spec/policy_machine_storage_adapters/active_record_spec.rb

```diff
@@ -92,7 +92,7 @@
             filters = { user_attributes: { color: 'purple' } }
 
             expect(
-              priv_pm.is_privilege_with_filters?(user_1, create, object_5, filters: filters)
+                priv_pm.is_privilege_with_filters?(user_1, create, object_5, filters: filters)
```

Comment:
Why did you make all these spacing changes? 2 spaces is the correct default.

Reply:
I used to think that auto-formatting is basic thing of any development environment in XXI century. I was wrong...